cmd/action/internal/update/update.go

```diff
@@ -0,0 +1,152 @@
+package update
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/pelletier/go-toml/v2"
+	"golang.org/x/mod/semver"
+)
+
+// Update logic:
+// 1. Check release file ~/.local/atlas/release.hcl for latest known release and poll time
+// 2. If last poll was less than 24h, poll GitHub public API https://docs.github.com/en/rest/reference/releases#get-the-latest-release
+// 3. Store the latest release metadata
+// 4. If current build Version, that is not development, is lower than the latest release, notify user
+
+type (
+	// Store is where the latest release data is stored on CLI host
+	Store struct {
```

Comment:
Name is ambiguous and doesn't mean anything obvious to me , something else ?

Reply:
I like the name, store as tokenStore, configStore, and part of update package is update.Store